src/key/opaque_key_nostd.rs

```diff
@@ -0,0 +1,168 @@
+
```

Comment:
Do we have to have two copies here? Seems to me like a lot is the same. Seems to me that having one file with all deviations behind a conditional compilation flag, would be easier to maintain.

Reply:
The imports are different one is `nostd` and one is `std`, I wasn't sure it seemed 1 file per target was easier then having a huge `#[cfg` head. I'll edit.

Reply:
I've changed my mind, and I guess we can leave as is.